apps/server/src/shared/testing/map-user-to-current-user.ts

```diff
@@ -1,15 +1,21 @@
 import { ObjectId } from '@mikro-orm/mongodb';
-
 import { Account, EntityId, User } from '@shared/domain';
 import { ICurrentUser } from '@src/modules/authentication';
 
-export const mapUserToCurrentUser = (user: User, account?: Account, systemId?: EntityId): ICurrentUser => {
-	const currentUser: Partial<ICurrentUser> = {
+export const mapUserToCurrentUser = (
+	user: User,
+	account?: Account,
+	systemId?: EntityId,
+	impersonated?: boolean
```

Comment:
What does this mean?

Reply:
Its a new flag introducted by philipp which marks the jwt as support jwt and the mapping missing here, so we added it